docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -80,7 +83,20 @@ home page](https://dash.metriport.com/). From here, take the following steps to
 </Tip>
 
 ---
-## 4. Register your Organization
+
+## 4. Integrate your app with Metriport
+
+Note that there are two ways to receive data from Metriport:
+
+- API: Your app requests data from Metriport:
+  - your application polls Metriport for Patients' medical data after initiating a query to fetch a Patient's documents;
+- Webhook: Metriport sends patient data to your app when it becomes available:
+  - your application exposes a webhook (endpoint), which Metriport calls when there's Patient data available after initiating a query;
+  - to learn more about how the Webhook flow works, see [our Webhooks guide](/home/api-info/webhooks).
```

Comment:
nit: I suggest we make Webhook the first option and mark it as the recommended way to integrate with Metriport